Backpack/Chip/Classes/BPKChip.h

```diff
@@ -23,6 +23,14 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+ * Enum values for specifying chip style
+ */
+typedef NS_ENUM(NSUInteger, BPKChipStyle) {
+    BPKChipStyleFilled = 0,
```

Comment:
These both need a doc comment describing them for them to be properly documented in the generated documentation site. You can test docs locally with

`bundle exec rake docs` followed by `cd docs && python -m http.server 3000` and then browsing to http://localhost:3000

Reply:
Added :)